index.html

```diff
@@ -14,67 +14,73 @@
 
 <body>
     <header>
-        <h1>Jiachun</h1>
+        <h1 aria-label="Jiachun" >Jiachun</h1>
         <nav>
-            <img class="header-mobile" src="./img/bar-icon.png" />
+            <img class="header-mobile" src="./img/bar-icon.png" alt="bar-icon" />
             <ul>
                 <li>Profolio</li>
                 <li>About</li>
                 <li>Contact</li>
-                <li><img class="header-desktop" src="./img/mail.png" /></li>
+                <li><img class="header-desktop" src="./img/mail.png" alt="mail-icon" /></li>
             </ul>
         </nav>
     </header>
 
```

Comment:
https://github.com/mpa-mxiang/profolio2/blob/7e82c2eb2f8d52c8e4e86a74b6e37b9f6d981ac7/index.html#L203

- [ ] For these kinds, you should specify what type of image they are not to cause ambiguity for screen readers and visually impaired persons 😃. for example "Project Image" or more explanatory like "Calculator Project Image" or "Google clone project snippet".

This is applicable to all ambiguous image `alt` attributes not just this 😉.